_commands/tools/nslookup.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+nslookup
+-------
+
+nslookup which stands for "name server lookup" is used to query internet name servers interactively for information. 
+
+~~~ bash
+nslookup [-option ...] [host-to-find | -[server]]
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+#### Example command
+~~~ 
+nslookup microsoft.com
+~~~
+##### Break it down
+The response would be: 
+
+Server:		141.211.125.17
+Address:	141.211.125.17#53
+
+Non-authoritative answer:
+Name:	microsoft.com
+Address: 23.100.122.175
+Name:	microsoft.com
+Address: 23.96.52.53
+Name:	microsoft.com
+Address: 191.239.213.197
+Name:	microsoft.com
+Address: 104.40.211.35
+Name:	microsoft.com
+Address: 104.43.195.251
```

Comment:
This doesn't look quite right. Did you preview it in your development environment? I would put this stdout in the code block as well.

![image](https://cloud.githubusercontent.com/assets/5882053/23089349/bb03b000-f544-11e6-8e45-53796d31b99a.png)